### ISSUE TITLE
Can't open blacklist

### ISSUE THREAD

**Issue by samuelrhaynes** [kboghdady/youTube_ads_4_pi-hole#19]
Hey mate, loving the script, however I think it's so large it causes the pihole admin to freeze if I try and load the blacklist

Anyone else have this issue? Any work arounds?

**Comment by SamboMambo** [comment]
I have this issue, i couldn't tell why it was freezing then i realized it's updating the list every hour

**Comment by DemonRok** [comment]
Hi @samuelrhaynes and @SamboMambo!

Freezeing is normal, blacklist contains more than 22,000 lines 💃 
Instead of using the youtube.sh script and running the blacklist every hour, I found much more convenient to insert the raw link of the file youtubelist.txt directly in the "Blocklists used to generate Pi-hole's Gravity" settings.
```
https://raw.githubusercontent.com/kboghdady/youTube_ads_4_pi-hole/master/youtubelist.txt
```
I have also modified crontab to perform Pi-hole's Gravity generation once a day as follows:
login as root or use sudo
```
sudo cd /etc/cron.d
sudo vi pihole
```
and modify the line 
```
35 4   * * *   root    PATH="$PATH:/usr/local/bin/" pihole updateGravity >/var/log/pihole_updateGravity.log || cat /var/log/pihole_updateGravity.log
```
so, every day at 4:35 (when I'm normally sleeping) crontab updates gravity.
In this way the blacklist wasn't touched.

As described in README.md more information about crontab are there: [https://crontab.guru/](https://crontab.guru/#35_4_*_*_*)
